frontend/src/components/pages/MultiUserBurndownChart/Presentation.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from "recharts";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import green from "@material-ui/core/colors/green";
+import red from "@material-ui/core/colors/red";
+
+const useStyles = makeStyles((theme) => ({
+  legend: {
+    padding: theme.spacing(1),
+  },
+  selectedFilter: {
+    color: "green",
+    marginRight: "1em"
+  },
+  unselectedFilter: {
+    marginRight: "1em"
+  }
+}));
+
+function Presentation({ userSnapshotArray, authedUser }){
+  const [metricFilter, setMetricFilter] = useState("TotalProjects");
```

Comment:
useState in a presentation file? Are you sure?

Reply:
Is it incorrect to use the useState hook in the Presentation file?